packages/react-graphql/src/hooks/mutation.ts

```diff
@@ -0,0 +1,29 @@
+import {OperationVariables} from 'apollo-client';
+import {MutationOptions} from 'react-apollo';
+import {DocumentNode} from 'graphql-typed';
+import {useMemo} from 'react';
+
+import {MutationHookOptions, MutationHookResult} from './types';
+import {useApolloClient} from './apollo-context';
+
+export default function useMutation<Data = any, Variables = OperationVariables>(
+  mutation: DocumentNode<Data, Variables>,
+  options: MutationHookOptions<Data, Variables> = {},
+): MutationHookResult<Data, Variables> {
+  const {client: overrideClient, ...mutationOptions} = options;
+
+  const client = useApolloClient(overrideClient);
+
+  const runMutation = useMemo(
+    () => (perMutationOptions?: MutationOptions<Data, Variables>) => {
```

Comment:
We don't mention `perMutationOptions` in the docs as far as I can tell, can we do so?